services/grpc/grpc.js

```diff
@@ -110,20 +111,43 @@ class ZapGrpc extends EventEmitter {
     this.on('subscribeGetInfo.data', async data => {
       const { synced_to_chain } = data
       if (synced_to_chain && !this.activeSubscriptions.channelgraph) {
-        this.unsubscribe('info')
+        // reduce poll interval to once a minute after synced_to_chain is true
+        await this.unsubscribe('info')
+        this.subscribe({ name: 'info', params: { pollInterval: 5000 } })
```

Comment:
comment says once per minute, but this is actually running every 5 seconds. I think once per minute is probably enough.

Reply:
sorry, reduced for testing and forgot to roll-back before force-pushing